clang/include/clang/Basic/DiagnosticParseKinds.td

```diff
@@ -1534,6 +1534,8 @@ def warn_omp51_compat_attributes : Warning<
   "standards before OpenMP 5.1">,
   InGroup<OpenMPPre51Compat>, DefaultIgnore;
 def err_omp_expected_colon : Error<"missing ':' in %0">;
+def err_omp_missing_comma : Error<
+  "missing ',' %select{after|in}0 %1">;
```

Comment:
Why need a selector if only first option is used?

Reply:
It's for any potential future use if someone wants to customize it to reuse it. Please let me know if you want me to remove it.

Reply:
Please, remove it for now

Reply:
Done, please take a look.